megaavr/cores/dxcore/wiring_analog.c

```diff
@@ -870,20 +870,51 @@ uint8_t digitalPinToTimerNow(uint8_t p) {
     }
   }
 */
-
 void analogReadSampleDelay(uint8_t delay) {
-  while (ADC0.COMMAND & ADC_STCONV_bm) {}
-  if (delay <= 15) {
-    ADC0.CTRLD &= 0b11110000;
-    ADC0.CTRLD |= delay;
-  }
+    while (ADC0.COMMAND & ADC_STCONV_bm) {}
```

Comment:
I would normally indent with 4 spaces but since the rest of DxCore seems to use 2 spaces of indentation we should probably stick to that for consistency.